### PR TITLE
OTC-530: Moving district is now blocked if local pricelists present.

### PR DESCRIPTION
**Pull request by wzglinieckisoldevelo**
https://openimis.atlassian.net/browse/OTC-530

Changes:
- When moving districts ask user if they want to do it
- Message also tells that this will detach local pricelists from health facilities
- Code will set pricelists foreign key to NULL in appropriate health facilities

### CODE REVIEW

**Comment by delcroip** [comment]
@wzglinieckisoldevelo  Jiri wanted to unlink the PL , can you change your warning to a cconfirmation box  : if ok move AND remove the conflicting pricelist if cancel BLOCK

br

**Comment by delcroip** [comment]
@still a draft ?

File: IMIS/MoveLocation.aspx.vb [hunk 1]
```diff
@@ -58,8 +60,13 @@ Public Class MoveLocation
...
         Try
-            AuditUserId = imisgen.getUserId(Session("User"))
             lblMsg.Text = ""
+            If IsPostBack = True Then
+                If Request.Params.Get("__EVENTARGUMENT").ToString = "MoveDelete" Then
+                    MoveDistrict(Request.Params.Get("__EVENTARGUMENT").ToString, Request.Params.Get("__EVENTARGUMENT").ToString)
+                End If
+            End If
+            AuditUserId = imisgen.getUserId(Session("User"))
             If IsPostBack = True Then Return
```
Comment (malinowskikam) [inline IMIS/MoveLocation.aspx.vb:70]: These two IF statements can be merged

**Review by malinowskikam — CHANGES_REQUESTED** [review]
@wzglinieckisoldevelo Please check my comment.

File: IMIS/MoveLocation.aspx.vb (hunk 1 above)
Comment (wzglinieckisoldevelo) [reply IMIS/MoveLocation.aspx.vb]: Request.Params exist only if IsPostBack is true, so merging would be problematic.

**Review by malinowskikam — APPROVED** [review]

**Review by delcroip — APPROVED** [review]

**Pull request merged**